gen.py

```diff
@@ -32,13 +33,19 @@
     argParser = argparse.ArgumentParser(
         prog="gen_daal4py.py",
         description=description,
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    argParser.add_argument('--daalroot', required=True,
-                           help="DAAL root directory (reads include dir in there)")
-    argParser.add_argument('--outdir', default='build',
-                           help="Output directory to store wrapper files to")
-    argParser.add_argument('--wall', default=False,
-                           action='store_true', help="Emit all warnings")
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    argParser.add_argument(
```

Comment:
@ahuber21, doesn't linter change camel to snake case for variables? I thought it's basic requirement for python linter

Reply:
That's new to me. Which linter is able to do that? Maybe we could add it to the pipeline